diff --git a/drivers/usb/misc/Kconfig b/drivers/usb/misc/Kconfig
index 9341cef76..e3391bf3e 100644
--- a/drivers/usb/misc/Kconfig
+++ b/drivers/usb/misc/Kconfig
@@ -314,7 +314,7 @@ config USB_ONBOARD_HUB
 
 config SPACEMIT_ONBOARD_USB_HUB
 	tristate "Spacemit onboard USB hub support"
-	depends on ARCH_SPACEMIT || COMPILE_TEST
+	depends on SOC_SPACEMIT
 	default SOC_SPACEMIT_K1X && USB_DWC3_SPACEMIT
 	help
 	  Say Y here if you want to support onboard usb hubs on Spacemit
